BUILD

```diff
@@ -0,0 +1,93 @@
+Building eudev
+===================
+
+The following instructions provide the steps needed to acquire and build the eudev
+software on the Debian, Devuan, and compatible Linux distributions.
+
+Please note that command examples in this document use the following convention. Commands
+prefixed by a "$" symbol are to be run as a regular, user. Commands with a "#" prefix
+need to be run as the root user. If your user has sudo access then sudo may be
+used instead of logging into the root account.
+
+
+1. Installing dependencies
+============================
+
+The first thing we need to do is install the dependencies required to build eudev. 
+This can be accomplished by running the following two commands:
+
+# apt-get update
+
+# apt-get install build-essential pkg-config docbook-xsl\
+          docbook-xml m4 automake autoconf intltool gperf\
+          gnu-efi libcap-dev libselinux1-dev libacl1-dev\
+          libkmod-dev libblkid-dev gobject-introspection\
+          libglib-object-introspection-perl xsltproc 
```

Comment:
add a space before the backslashes (stylistic thing)

is it preferable to use apt over apt-get these days?

Reply:
I've updated the formatting on the install command.

Regarding apt vs apt-get, the two do exactly the same thing. Which one to use is a person preference. People often suggest apt for interactive use and apt-get for scripts, but they are functionally the same.